microservices/tcp_validator/index.js

```diff
@@ -63,6 +63,7 @@ const handleTCPVerification = async (uid, url, port, writeText) => {
   } catch (e) {
     console.error("Hubo Error ", e);
   } finally {
+    console.log(response)
```

Comment:
@MatiasMinchaca please remove this console log because is not be part of the requirement